docs/debugging/pydev.rst

```diff
@@ -111,7 +111,7 @@ Running/Debugging Pyramid under Pydev
    ``$WORKSPACE`` is the name of the PyDev workspace containing your project
    
    To create a working example, copy the `pyramid tutorial step03 
-   <https://pyramid_tutorials.readthedocs.org/en/latest/getting_started/03-config/index.html>`_
+   <https://pyramid_tutorials.readthedocs.io/en/latest/getting_started/03-config/index.html>`_
```

Comment:
This URL is not valid. We moved this tutorial out of pyramid_tutorials into supported Pyramid docs as the Quick Tutorial. Use http://docs.pylonsproject.org/projects/pyramid/en/latest/quick_tutorial/ini.html

Note: whenever you change a URL in docs, you *must* validate it.